recirq/quantum_chess/quantum_board.py

```diff
@@ -255,13 +279,15 @@ def sample_with_ancilla(self, num_samples: int
                         f'Discarded {error_count} from error mitigation '
                         f'{noise_count} from noise and '
                         f'{post_count} from post-selection\n')
+                    self.record_time("sample_with_ancilla", t0)
                     return (rtn, ancilla)
         else:
             rtn = [self.state] * num_samples
             self.debug_log += (
                 f'Discarded {error_count} from error mitigation '
                 f'{noise_count} from noise and {post_count} from post-selection\n'
             )
+        self.record_time("sample_with_ancilla", t0)
```

Comment:
Let's use single quotes for consistency.

Reply:
Thanks! It's changed.